src/mount/sauna_client.cc

```diff
@@ -224,6 +224,59 @@ static std::atomic<bool> gDirectIo(false);
 static uint32_t lock_request_counter = 0;
 static std::mutex lock_request_mutex;
 
+#ifdef _WIN32
```

Comment:
I understand the idea, and look good enough for first version, but in the future we need to avoid implementation specifics in this part of the code. Probably the idea will be to convert the client to a class, and just extend it, but this mixture is strange and can led to extensibility issues, for instance if we plan to have another windows client based in a different library.

Reply:
Understood.